prboom2/doc/umapinfo.txt

```diff
@@ -307,10 +307,119 @@ Thingtypes:
     PlasmaBall2
     EvilSceptre
     UnholyBible
+    MusicChanger
+    Deh_Actor_145
```

Comment:
Sorry for nagging you again, but I guess for documentation purposes it will be sufficient to abbreviate these with 
```
Deh_Actor_145
...
Deh_Actor_249
```

Reply:
Done!